toplevel/genprintval.ml

```diff
@@ -27,7 +26,9 @@ module type OBJ =
   sig
     type t
     val repr : 'a -> t
-    val obj : t -> 'a
+    (* [base_obj] assumes that the value has a marshallable base type. *)
+    val base_obj : t -> 'a
```

Comment:
I have the feeling that `base_obj` is not an ideal name but with the type it does convey the between the difference between `obj` and `base_obj`.